lib/_debug_agent.js

```diff
@@ -122,7 +122,7 @@ Client.prototype._transform = function _transform(data, enc, cb) {
       if (!/\r\n/.test(this.buffer))
```

Comment:
This could become a this.buffer.contains() test? I don't know about its performance, but would be in the same spirit of this PR.